MyApplication/app/src/main/res/layout/activity_constraint.xml

```diff
@@ -0,0 +1,144 @@
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+
+    <ImageView
+        android:id="@+id/image_1"
+        android:layout_width="100dp"
+        android:layout_height="120dp"
+        android:src="@drawable/man"
+        android:contentDescription="@string/man_image1"
+        app:layout_constraintEnd_toStartOf="@+id/image_2"
+        app:layout_constraintHorizontal_chainStyle="spread"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/backgroud_image" />
+
+    <ImageView
+        android:id="@+id/image_2"
+        android:layout_width="100dp"
+        android:layout_height="120dp"
+        android:src="@drawable/man"
+        android:contentDescription="@string/man_image2"
+        app:layout_constraintEnd_toStartOf="@+id/image_3"
+        app:layout_constraintStart_toEndOf="@+id/image_1"
+        app:layout_constraintTop_toBottomOf="@+id/backgroud_image" />
+
+    <ImageView
+        android:id="@+id/image_3"
+        android:layout_width="100dp"
+        android:layout_height="120dp"
+        android:contentDescription="@string/man_image3"
+        android:src="@drawable/man"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toEndOf="@+id/image_2"
+        app:layout_constraintTop_toBottomOf="@+id/backgroud_image" />
+
+    <TextView
+        android:id="@+id/sub_bullet_1"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="50dp"
+        android:text="1.Harroy potter"
+        android:textSize="25dp"
+        app:layout_constraintStart_toEndOf="@+id/sub_ponit_guideline"
+        app:layout_constraintTop_toBottomOf="@+id/image_1" />
+
+    <TextView
+        android:id="@+id/bullet_2"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="20dp"
+        android:text="2.Granger"
+        android:textSize="25dp"
+        app:layout_constraintStart_toStartOf="@id/sub_ponit_guideline"
+        app:layout_constraintTop_toBottomOf="@+id/sub_bullet_1" />
+
+    <TextView
+        android:id="@+id/bullet_3"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="20dp"
+        android:text="3.Ron Weasly"
+        android:textSize="25dp"
+        app:layout_constraintStart_toStartOf="@id/sub_ponit_guideline"
+        app:layout_constraintTop_toBottomOf="@+id/bullet_2" />
+
+
+    <TextView
+        android:id="@+id/subject_master"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="100dp"
+        android:text="subject mastered"
+        android:textSize="30dp"
+        app:layout_constraintStart_toStartOf="@id/backgroud_image"
+        app:layout_constraintTop_toBottomOf="@id/bullet_3" />
+
+    <TextView
```

Comment:
Please read the assignment carefully. you have to use `ScrollView` to show at least 10 subjects here.